doc/releases/release-notes-3.3.rst

```diff
@@ -401,6 +401,7 @@ Libraries / Subsystems
   * Added new API call `fs_mkfs`.
   * Added new sample `samples/subsys/fs/format`.
   * FAT FS driver has been updated to version 0.15 w/patch1.
+  * Added FAT FS reentrant Kconfig :kconfig:option:`CONFIG_FS_FATFS_REENTRANT`.
```

Comment:
The Kconfig is not reentrant. Rephrase this to be more clear that the Koconfig enables reentrant support in FAT FS.